server/models/questions/getPublicContestQuestions.js

```diff
@@ -0,0 +1,43 @@
+const { pool } = require('../database')
+
+/**
+ * @param {*} param0
+ * @param {Number} param0.cursor
+ * @param {Number} param0.search
+ * @param {Number} param0.limit
+ * @return {Promise}
+ */
+
+function getPublicContestQuestions({ cursor, limit, search }) {
+  return new Promise((resolve, reject) => {
+    let query = `SELECT * FROM contests c
+                INNER JOIN contests_questions cq ON c.id=cq.contest_id
+                INNER JOIN questions q ON cq.question_id=q.id WHERE `
+    const qArr = []
+
+    if (cursor) {
+      query += `q.id>? AND `
+      qArr.push(+cursor)
+    }
+
+    query += `c.end_time <= NOW() `
```

Comment:
In most of the places, we have used the date-time of nodejs server. But it appears that we should change those to MySQL time only. That will ease out many queries . For eg. getContests() API

Reply:
Yes..it will ease out all inserts as well..using NOW()  or CURRENT_TIMESTAMP